### PR TITLE
Add bind checker daemon

### PR DESCRIPTION
**Pull request by pablomendezroyo**
Add bind checker daemon DOCKER network config

### CODE REVIEW

File: packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  const conflictingContainer = await findContainerByIP(network, params.BIND_IP);
+  if (conflictingContainer) {
+    logs.info(
+      `address ${params.BIND_IP} already in used by ${conflictingContainer.Name}, freeing it`
+    );
+    await network.disconnect({ Container: conflictingContainer.Name });
+  }
+  logs.info(`Starting ${params.bindContainerName} container`);
+  // The docker compose will start the container with the right IP
+  await dockerComposeUp(getDockerComposePath(params.bindDnpName, true));
```
Comment (dappnodedev) [inline packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts:22]: Here are we 100% sure that `ensureBindComposeIp` has been executed?

File: packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    await docker.getContainer(params.bindContainerName).inspect()
+  ).State.Running;
+
+  // check if the bind container is running
+  if (!isBindRunning) {
+    logs.info(`${params.bindContainerName} container is not running`);
+    await disconnectConflictingContainerAndStartBind();
+  } else {
+    // check is connected to dncore_network
+    const isConnectToNetwork = Object.values(
```
Comment (dappnodedev) [inline packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts:45]: Small detail: I would call this `isConnectedToNetwork`

File: packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  docker,
+  findContainerByIP,
+  dockerComposeUp,
+} from "@dappnode/dockerapi";
+import { logs } from "@dappnode/logger";
+import { params } from "@dappnode/params";
+import { getDockerComposePath } from "@dappnode/utils";
+import Dockerode from "dockerode";
+
+async function disconnectConflictingContainerAndStartBind(): Promise<void> {
```
Comment (dappnodedev) [inline packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts:11]: You could use `disconnectConflictingContainerIfAny` inside this function, right?

**Review by dappnodedev — CHANGES_REQUESTED** [review]

File: packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts (hunk 2 above)
Comment (pablomendezroyo) [reply packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts]: fixed!

File: packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts (hunk 1 above)
Comment (pablomendezroyo) [reply packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts:22]: yes!

File: packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts (hunk 3 above)
Comment (pablomendezroyo) [reply packages/daemons/src/bind/ensureBindContainerIpAndRunning.ts:11]: re used!

**Review by dappnodedev — APPROVED** [review]

**Pull request merged**